frontend/src/app/error.tsx

```diff
@@ -11,7 +11,7 @@ import { RequestFailedError } from "@/lib/api"
 
 type ErrorPageProps = {error: Error, reset: () => void };
 
-function NetworkErrorPage({ error, reset }: ErrorPageProps) {
+function NetworkErrorPage({ error, reset: _reset }: ErrorPageProps) {
```

Comment:
Is there any reason for destructuring the `reset` field of `ErrorPageProps` when it's not used?

Reply:
it's used in `UnexpectedErrorPage` further down, and both take the same props

Reply:
But we only need to destructure the props that we use, right? Or is there any magic happening?

Reply:
ah not sure, guess I misunderstood